src/System.Runtime.Extensions/tests/System/AppDomainTests.netcoreapp.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public partial class AppDomainTests
+    {
+        [Fact]
+        public void GetSetupInformation()
+        {
+            // The behaviour is different from full framework due to the https://github.com/dotnet/corefx/issues/23063.
+            // We can modify this test later to check if the TargetFrameworkName starts with .NETCore or .NETFramework.
```

Comment:
Note that the target framework is not the framework that the process is actually running on. It is the value read from the TargetFrameworkAttribute on the .exe that started the process - xunit driver in this case. If we ever change this test like you have suggested here, we are going to be testing how to xunit driver was built. I do not see a problem with it, but it may be worth a comment at that point.

Reply:
I will keep in mind to add the comment when we will modify the test in future.